backend-peer/src/lib/services/cloudagent.ts

```diff
@@ -46,19 +50,24 @@ interface GetConnectionParams {
   theirDid?: string
   theirLabel?: string
 }
+type ImplicitInvitationResponse = {
+  outOfBandRecord: OutOfBandRecord
+  connectionRecord: ConnectionRecord
+}
 
 /**
  * @example "821f9b26-ad04-4f56-89b6-e2ef9c72b36e"
  */
 export type RecordId = string
-
+@injectable()
 @singleton()
 export class CloudagentManager {
   private url_prefix: string
+  private env: Env
 
-  constructor() {
-    const env = container.resolve<Env>('env')
-    this.url_prefix = `http://${env.CLOUDAGENT_HOST}:${env.CLOUDAGENT_PORT}`
+  constructor(private eventEmitter: NiceEventEmitter) {
+    this.env = container.resolve<Env>('env')
```

Comment:
Can this `env: Env` not also be injected in the constructor params?

Reply:
it kept throwing errors if I would not use specifically `container.resolve<Env>('env')`